cmd/collectors/ems/ems.go

```diff
@@ -634,6 +643,8 @@ func (e *Ems) HandleResults(result []gjson.Result, prop map[string][]*emsProp) (
 								Msg("Unable to find metric")
 						}
 					}
+					instanceLabelCount = +instanceLabelCountPs
```

Comment:
did you mean to do `instanceLabelCount += instanceLabelCountPs`

Reply:
yes, typo.